draft-ietf-taps-impl.md

```diff
@@ -327,31 +327,29 @@ Branching for derived endpoints is the final step, and may have multiple layers
+   * Low Latency/Interactive:
      Prefer paths with the lowest expected Round Trip Time
-   * Constant Rate:
+   * Constant-Rate Streaming:
      Prefer paths that can satisfy the requested Stream Send or Stream Receive Bitrate, based on observed maximum throughput
-   * Scavenger/Bulk:
-     Prefer paths with the highest expected available bandwidth, based on observed maximum throughput
-
-\[Note: See {{branch-sorting-non-consensus}} for additional examples related to Properties under discussion.]
 
 Implementations should process properties in the following order: Prohibit, Require, Prefer, Avoid.
-If Protocol or Path Selection Properties contain any prohibited properties, the implementation should first purge branches containing nodes with these properties. For required properties, it should only keep branches that satisfy these requirements. Finally, it should order branches according to preferred properties, and finally use avoided properties as a tiebreaker.
+If Selection or Connection Properties contain any prohibited properties, the implementation should first purge branches containing nodes with these properties. For required properties, it should only keep branches that satisfy these requirements. Finally, it should order branches according to preferred properties, and finally use avoided properties as a tiebreaker.
 
-For Require and Avoid, Path Selection Properties take precedence over Protocol Selection Properties.
-For example, if the application has indicated both a preference for WiFi over LTE and for a feature only available in SCTP, branches will be first sorted accord to the Path Selection Property, with WiFi at the top. Then, branches with SCTP will be sorted to the top within their subtree according to the Protocol Selection Property. However, if the implementation has cached the information that SCTP is not available on the path over WiFi, there is no SCTP node in the WiFi subtree. Here, the path over WiFi will be tried first, and, if connection establishment succeeds, TCP will be used. So the Path Selection Property of preferring WiFi takes precedence over the Protocol Selection Property of preferring SCTP.
+For Require and Avoid, path selection takes precedence over protocol selection.
+For example, if the application has indicated both a preference for WiFi over LTE and for a feature only available in SCTP, branches will be first sorted accord to path selection, with WiFi at the top. Then, branches with SCTP will be sorted to the top within their subtree according to the properties influencing protocol selection. However, if the implementation has cached the information that SCTP is not available on the path over WiFi, there is no SCTP node in the WiFi subtree. Here, the path over WiFi will be tried first, and, if connection establishment succeeds, TCP will be used. So the Selection Property of preferring WiFi takes precedence over the Property that led to a preference for SCTP.
 
```

Comment:
This example should actually be moved to the previous paragraph where the  branchingorder is discussed or deleted.

Reply:
Sorry, my comment was messed up. I meant to say that the example should move to the previous section (not paragraph), where the branching order is defined. It is not connected to Require and Avoid so that sentence can be removed.

Reply:
understood, done